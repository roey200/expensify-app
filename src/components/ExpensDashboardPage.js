import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'
import ExpensesSummary from './ExpensesSummary'

const ExpensDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <ExpenseListFilters />
        <ExpenseList />
    </div>
)

export default ExpensDashboardPage