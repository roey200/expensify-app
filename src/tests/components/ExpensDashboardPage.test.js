import React from 'react';
import { shallow } from 'enzyme';
import ExpensDashboardPage from '../../components/ExpenseListItem';

test('should render ExpensDashboardPage correctly', () => {
  const wrapper = shallow(<ExpensDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
