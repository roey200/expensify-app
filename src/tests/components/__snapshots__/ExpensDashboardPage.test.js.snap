// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensDashboardPage correctly 1`] = `
<div>
  <[object Object]
    to="/edit/undefined"
  >
    <h3 />
  </[object Object]>
  <p>
    $0.00
    -
    January 1st, 1970
  </p>
</div>
`;
